gitgud/skills/rampup/test_levels.py

```diff
@@ -4,30 +4,8 @@
 
 from . import skill
 
-
 level_tests = [
-    (
-        # User would use something like "git checkout a1b2c3d4"
-        skill['detaching'], [
-            'git checkout bugFix',
-            'git checkout @^',
-            'git checkout @{1}',
-        ]
```

Comment:
Here, you're going to need to get the solution from the method that the level produces to get to the solution

Reply:
See here: https://github.com/benthayer/git-gud/pull/187#discussion_r435473464